tests/run-pass/time.rs

```diff
@@ -4,15 +4,19 @@
 use std::time::{SystemTime, Instant};
 
 fn main() {
+
     let now1 = SystemTime::now();
     // Do some work to make time pass.
     for _ in 0..10 { drop(vec![42]); }
     let now2 = SystemTime::now();
     assert!(now2 > now1);
 
-    let now1 = Instant::now();
-    // Do some work to make time pass.
-    for _ in 0..10 { drop(vec![42]); }
-    let now2 = Instant::now();
-    assert!(now2 > now1);
+    // Shims not yet implemented on Windows
+    #[cfg(not(windows))] {
```

Comment:
Yes, looking good. :)
This is also fine for the PR; if you plan to implement `Instant::now()`, please make that a separate PR.